public/js/portfolio.js

```diff
@@ -48,9 +48,20 @@ let portfolio = createPage(
             })();
         }
     },
-    function () {
-        this.$nextTick(() => this.search());
-    }
+    {
+        updated: function() {
+            this.$nextTick(() => this.search());
+        },
+        activated: function(){
+            console.log('Activated!')
+        },
+        deactivated: function(){
+            console.log('Deactivated!')
+        }
+    },
+    [],
+    {},
+    null,
```

Comment:
Could this change on portfolio.js be reverted and keep working?

Reply:
Yep, I just removed those from the template on protfolio.js.